dashboard/static/js/Roles.js

```diff
@@ -47,6 +46,17 @@ $(document).ready(function(){
                 table.rows[index + 1].cells[1].innerHTML = role["name"]//name
                 table.rows[index + 1].cells[2].innerHTML = role['description'];//description
                 
+                groups_amt = role['groups'].length
+                groups = ""
+                for(var index2 = 0; index2 < groups_amt; index2++){
+                    
+                    groups +=  "<button type='button' class='btn btn-primary btn-xs'>" + role['groups'][index2].name + " </button> "
```

Comment:
Can you use jquery to create the element and edit it instead? Try using this kind of syntax: [jquery docs](http://api.jquery.com/jQuery/#jQuery2) or [our code](https://github.com/CSUMB-SP17-CST499/group-assignment-tool/blob/111917e449c6402a18bf1b94a827461660aba1d4/dashboard/static/js/RoleDropDown.js) has an example in the `displayRolesByName` function.